examples/simulation/Tutorial1/macros/CMakeLists.txt

```
@@ -51,14 +51,14 @@ foreach(mcEngine IN LISTS mcEngine_list)
   )
 endforeach()
 
-add_test(NAME ex_tutorial1_Geant4MT
-         COMMAND ${CMAKE_CURRENT_BINARY_DIR}/run_tutorial1.sh 10 \"TGeant4\" true false)
+add_test(NAME ex_sim_tutorial1_Geant4MT
+         COMMAND ${CMAKE_CURRENT_BINARY_DIR}/run_tutorial1.sh 20 \"TGeant4\" true false)
 set_tests_properties(ex_tutorial1_Geant4MT PROPERTIES
   TIMEOUT ${maxTestTime}
   PASS_REGULAR_EXPRESSION "Simulation successful."
 )
 
-add_test(NAME ex_tutorial1_PostInitConfig
+add_test(NAME ex_sim_tutorial1_PostInitConfig
          COMMAND ${CMAKE_CURRENT_BINARY_DIR}/run_tutorial1.sh 1 \"TGeant4\" false true)
 set_tests_properties(ex_tutorial1_PostInitConfig PROPERTIES
```

You should rename the other occurences as well, please.

```suggestion
set_tests_properties(ex_sim_tutorial1_PostInitConfig PROPERTIES
```

---

Oh, you just did.

Resolving.